client/src/lib/components/SessionProgress/SessionProgress.tsx

```diff
@@ -41,8 +41,8 @@ type SessionProgressProps = {
 const SessionProgress: React.FC<SessionProgressProps> = ({items}) => {
   const colors = useMemo(
     () => [
-      hexToRgba(COLORS.MEDIUM_GREEN, 1),
-      hexToRgba(COLORS.MEDIUM_GREEN, 1),
+      hexToRgba(COLORS.MEDIUM_DARK_GREEN, 1),
+      hexToRgba(COLORS.MEDIUM_DARK_GREEN, 1),
```

Comment:
Does this have the same name in Figma? Just so we're still aligned with the design system